### ISSUE TITLE
Add a flag CFP

### ISSUE THREAD

**Issue by aclerbois** [daveaglick/discoverdotnet#169]
Hello, 

I think about create a flag for Call4Paper in the events section with a link to this CFP. 

What's do you think about this idea? 

Best regards,

Adrien Clerbois

**Comment by gep13** [comment]
This sounds like a great idea to me!

**Comment by daveaglick** [comment]
Yeah, this is a fantastic idea. When we could have a page listing all the events seeking submissions and how to submit to them.

**Comment by matkoch** [comment]
Anyone working on adding this?

**Comment by japierdolewszystkozajete27** [comment]
Shoudn't we first work on hiding past events? 
Also - great idea! 🥇
